src/components/button.js

```diff
@@ -0,0 +1,8 @@
+import React from "react";
```

Comment:
rename this file to be the name of the component.
We use CamleCase.js for react components.
(other people use .jsx but not us, cause .js is good)
```ClickCounter.js```
```button.js`